Tests/PostgresNIOTests/New/PSQLRowStreamTests.swift

```diff
@@ -19,20 +19,29 @@ final class PSQLRowStreamTests: XCTestCase {
         
         XCTAssertEqual(try stream.all().wait(), [])
         XCTAssertEqual(stream.commandTag, "INSERT 0 1")
+
+        XCTAssertNoThrow(try stream.drain().wait())
     }
-    
+
     func testFailedStream() {
         let stream = PSQLRowStream(
             source: .noRows(.failure(PSQLError.serverClosedConnection(underlying: nil))),
             eventLoop: self.eventLoop,
             logger: self.logger
         )
-        
+
+        let expectedError = PSQLError.serverClosedConnection(underlying: nil)
+
         XCTAssertThrowsError(try stream.all().wait()) {
-            XCTAssertEqual($0 as? PSQLError, .serverClosedConnection(underlying: nil))
+            XCTAssertEqual($0 as? PSQLError, expectedError)
+        }
+
+        // Drain should work
+        XCTAssertThrowsError(try stream.drain().wait()) {
+            XCTAssertEqual($0 as? PSQLError, expectedError)
```

Comment:
in case of error, drain will just throw the same error.
Though realistically if there is an error users will not reach this far in the code.